.github/workflows/ci.yml

```diff
@@ -252,6 +264,13 @@ jobs:
             ctest . -j2 --output-on-failure
           fi
 
+      - name: Run lcov
+        if: matrix.build == 'coverage'
+        run: |
+          lcov --directory ./app-build/ --capture --output-file ./app-build/app_coverage --gcov-tool gcov-10
+          lcov --remove ./app-build/app_coverage '/usr/*' "${HOME}"'/.cache/*' '*/lib/*' --output-file ./app-build/app_coverage
```

Comment:
I think excludes should be 
* `/usr/*`
* `${{ github.workspace }}/app/lib/*`
* `${{ github.workspace }}/app-build/vendor/*`

I don't think we need `HOME` and `.cache`

Reply:
ah okay, we just copied it from the example and added the `*/lib/*`